build_tools/cmake/iree_python.cmake

```diff
@@ -228,11 +232,10 @@ function(iree_py_library)
     )
   endforeach()
 
-  # Add PYEXT_DEPS.
-  if(${ARG_PYEXT_DEPS})
-    foreach(V ${IREE_MULTIPY_VERSIONS_EFFECTIVE})
-      add_dependencies(${_NAME} ${ARG_PYEXT_DEPS})
-    endforeach()
+  # Add PYEXT_DEPS if any.
+  if(ARG_PYEXT_DEPS)
+    list(TRANSFORM ARG_PYEXT_DEPS REPLACE "^::" "${_PACKAGE_NS}::")
+    add_dependencies(${_NAME} ${ARG_PYEXT_DEPS})
```

Comment:
This makes sense to me, but I'm worried about altering this multipy stuff that I don't understand. @stellaraccident ?

Reply:
Any multipy stuff remaining survived the delete by accident. Everything load bearing about it is gone and i may have missed this.

Reply:
Well good because this didn't make any sense to me :grin: